src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,222 @@
+        assert ret == 'Success'
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'proxy_pam_target': 'sssdproxyldap',
+                         'id_provider': 'proxy',
+                         'proxy_lib_name': 'ldap',
+                         'auth_provider': 'proxy'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        execute_cmd(multihost, "id foo1")
+        for error_error in ['[sysdb_set_entry_attr] (6): '
+                            'Error: 14 (Bad address)',
+                            '[sysdb_store_user] (6): '
+                            'Error: 14 (Bad address)',
+                            'Internal Error (Cannot make/remove '
+                            'an entry for the specified session)']:
+            with pytest.raises(subprocess.CalledProcessError):
+                execute_cmd(multihost, f"grep {error_error} /var/log/sssd/*")
+        del_member = [(ldap.MOD_REPLACE, 'loginShell',
+                       "/bin/bash".encode('utf-8'))]
+        (ret, _) = ldap_inst.modify_ldap(user_dn, del_member)
+        assert ret == 'Success'
+        # non existing netgroup returned with proxy provider
+        # when proxy lib name is ldap bz804103
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "getent netgroup testsumgroup")
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'proxy_pam_target': 'sssdproxyldap',
+                         'id_provider': 'proxy',
+                         'proxy_lib_name': 'ldap',
+                         'auth_provider': 'proxy'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        execute_cmd(multihost, "getent netgroup netgroup_1")
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "getent netgroup testsumgroup")
+
+    def test_bz647816(self, multihost, backupsssdconf):
+        """
+        :title: More than 10 auth attempt times out bz647816
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=647816
+        :id: 201a44c0-b252-11ec-94b7-845cf3eff344
+        :steps:
+          1. Auth a user more than 12 times
+          2. Search for Error
+        :expectedresults:
+          1. Should succeed
+          2. Should not succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "getent netgroup testsumgroup")
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'proxy_pam_target': 'sssdproxyldap',
+                         'id_provider': 'proxy',
+                         'proxy_lib_name': 'files',
+                         'auth_provider': 'proxy'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        execute_cmd(multihost, "echo 'QAeng    "
```

Comment:
This is for a different BZs (bz804103/bz801377?) and should have tests in their own.

Reply:
done